downstream/assemblies/platform/assembly-configure-aap-operator.adoc

```diff
@@ -0,0 +1,27 @@
+ifdef::context[:parent-context: {context}]
+
+[id="configure-aap-operator_{context}"]
+
+:context: configure-aap-operator
+
+= Configuring the {PlatformName} operator on {OCP}
+
+The central configuration for {PlaformNameShort} allows you to manage the following {PlatformNameShort} components form a single user interface:
+
+* {ControllerNameStart}
+* {HubNameStart}
+* {EDAName}
+* {LightspeedShortName} (This feature is disabled by default, you must opt in to use it.)
+
+Before you can deploy the central configuration you need to have {PlatformNameShort} operator installed in a namespace. 
+If you have not installed {PlatformNameShort} operator see <<Installing the Red Hat Ansible Automation Platform operator on Red Hat OpenShift Container Platform>>.
+
+If you have the {PlatformNameShort} operator and some or all of the {PlatformNameShort} components installed see <<Deploying your central configuration with existing {PlatformNameShort} components>> for how to proceed. 
```

Comment:
Should "{PlatformNameShort} operator" be "{OperatorPlatform}"? Think there's a few instances of this in this PR.